src/aap_eda/api/vault.py

```diff
@@ -46,17 +46,27 @@ def encrypt_string(password: str, plaintext: str, vault_id: str) -> str:
     child.expect("Reading plaintext input from stdin*")
     child.sendline(plaintext)
     child.sendcontrol("D")
-    i = child.expect(["Encryption successful", "ERROR"])
-    if i == 0:
-        child.readline()
-        return "".join(
-            line.decode().lstrip()
-            for line in child
-            if not line.decode().startswith("!vault")
-        )
+
+    encrypted_successful_found = False
```

Comment:
@hsong-rh can you give some examples of why this was needed with the different versions of ansible-core.

Reply:
If it depends on versions of ansible-core we may want to enforce a version. We don't expect a version lower than 2.16